src/components/ChannelListItemComponent.js

```diff
@@ -34,9 +34,34 @@ class ChannelListItemComponent extends React.Component {
     this.setState({ openClose: !openClose });
   };
 
+  getSdkStatusName(status) {
```

Comment:
why not just using the sdk status? it is just a human readable name ?

Reply:
basically yeah, but i think is prettier to use a word rather than a enum name, for example CHANNEL_SETTLED is not pretty but Settled it is so basically that's why

Reply:
Anyway it was like that from the start, i didn't want to change that